Sources/DangerShellExecutor/ShellExecutor.swift

```diff
@@ -35,24 +51,26 @@ public struct ShellExecutor: ShellExecuting {
 
     public func execute(_ command: String,
                         arguments: [String],
-                        environmentVariables: [String: String]) -> String {
-        let task = makeTask(for: command, with: arguments, environmentVariables: environmentVariables)
+                        environmentVariables: [String: String],
+                        outputFile: String?) -> String {
+        let task = makeTask(for: command, with: arguments, environmentVariables: environmentVariables, outputFile: outputFile)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerShellExecutor/ShellExecutor.swift;
  Line: 56;
-->

- :warning: Line should be 120 characters or less: currently 126 characters (`line_length`)